src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java

```diff
@@ -153,19 +150,79 @@ public SELF isBlank() {
   }
 
   /**
-   * Verifies that the actual {@code CharSequence} is not blank, i.e. either is {@code null}, empty or
-   * contains at least one non-whitespace characters.
+   * Verifies that the actual {@code CharSequence} is:
+   * <ul>
+   *   <li>not {@code null}</li>
+   *   <li>not empty</li>
+   *   <li>contains at least one non-whitespace character</li>
+   * </ul>
    * <p>
    * It uses the same whitespace definition as the {@link #isBlank()} assertion.
```

Comment:
add a note saying that the behavior has changed and that the original behavior is now in `containsNotOnlyWhitespaces`

Reply:
Done